api/v1alpha1/ratelimit_types.go

```diff
@@ -220,20 +220,22 @@ type SourceMatch struct {
 }
 
 // HeaderMatch defines the match attributes within the HTTP Headers of the request.
-type HeaderMatch struct { // TODO: zhaohuabing this type could be replaced with a general purpose StringMatch type.
+type HeaderMatch struct {
 	// Type specifies how to match against the value of the header.
 	//
 	// +optional
 	// +kubebuilder:default=Exact
 	Type *HeaderMatchType `json:"type,omitempty"`
 
 	// Name of the HTTP header.
+	// The header name is case-insensitive unless PreserveHeaderCase is set to true.
+	// For example, "Foo" and "foo" are considered the same header.
+	//
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:MaxLength=256
 	Name string `json:"name"`
 
-	// Value within the HTTP header. Due to the
-	// case-insensitivity of header names, "foo" and "Foo" are considered equivalent.
```

Comment:
This existing comment seems to be in wrong place. Moving it to the header comment.